setup.cfg

```diff
@@ -2,4 +2,4 @@
 test=pytest
 
 [tool:pytest]
-rootdir=tests
```

Comment:
I deleted `rootdir` since it no used to determinate the roodir:

> look for pytest.ini, tox.ini and setup.cfg files in the ancestor directory and upwards. If one is matched, it becomes the ini-file and its directory becomes the rootdir.

From : https://docs.pytest.org/en/3.0.0/customize.html#initialization-determining-rootdir-and-inifile